src/biokbase/narrative/widgetmanager.py

```diff
@@ -585,6 +585,8 @@ def show_data_widget(self, upa, title=None, cell_id=None, tag="release"):
                 }
             }
         else:
+            if len(type_spec['view_method_ids']) == 0:
```

Comment:
What comes back from the service manager if "view_method_ids" is absent from the spec? Do we want to support that case as well?